cmd/vpu_plugin/README.md

```diff
@@ -47,86 +38,38 @@ This card has:
-```
-
-### Deploying as a DaemonSet
-
-To deploy the vpu plugin as a daemonset, you first need to build a container image for the
-plugin and ensure that is visible to your nodes.
-
-#### Build the plugin image
-
-The following will use `docker` to build a local container image called
-`intel/intel-vpu-plugin` with the tag `devel`.
-
-The image build tool can be changed from the default `docker` by setting the `BUILDER` argument
-to the [`Makefile`](/Makefile).
-
-```bash
-$ cd $(go env GOPATH)/src/github.com/intel/intel-device-plugins-for-kubernetes
-$ make intel-vpu-plugin
-...
-Successfully tagged intel/intel-vpu-plugin:devel
-```
-
-#### Deploy plugin DaemonSet
-
-You can then use the [example DaemonSet YAML](/deployments/vpu_plugin/base/intel-vpu-plugin.yaml)
-file provided to deploy the plugin. The default kustomization that deploys the YAML as is:
-
-```bash
-$ kubectl apply -k deployments/vpu_plugin
+$ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/vpu_plugin?ref=<RELEASE_VERSION>
 daemonset.apps/intel-vpu-plugin created
 ```
 
-> **Note**: It is also possible to run the VPU device plugin using a non-root user. To do this,
-the nodes' DAC rules must be configured to device plugin socket creation and kubelet registration.
-Furthermore, the deployments `securityContext` must be configured with appropriate `runAsUser/runAsGroup`.
+Where `<RELEASE_VERSION>` needs to be substituted with the desired release version or `main`.
 
 For xlink device, deploy DaemonSet as
 ```bash
-$ kubectl apply -k deployments/vpu_plugin/overlays/xlink
+$ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/vpu_plugin/overlays/xlink
 daemonset.apps/intel-vpu-plugin created
 ```
 
-### Deploy by hand
-
-For development purposes, it is sometimes convenient to deploy the plugin 'by hand' on a node.
-In this case, you do not need to build the complete container image, and can build just the plugin.
-
-#### Build the plugin
-
-First we build the plugin:
-
-> **Note:** this vpu plugin has dependency of libusb-1.0-0-dev, you need install it before building vpu plugin
-
-```bash
-$ cd $(go env GOPATH)/src/github.com/intel/intel-device-plugins-for-kubernetes
-$ make vpu_plugin
-```
-
-#### Run the plugin as administrator
+Nothing else is needed. See [the development guide](../../DEVEL.md) for details if you want to deploy a customized version of the plugin.
 
-Now we can run the plugin directly on the node:
-
-```bash
-$ sudo $(go env GOPATH)/src/github.com/intel/intel-device-plugins-for-kubernetes/cmd/vpu_plugin/vpu_plugin
-VPU device plugin started
-```
+> **Note**: It is also possible to run the VPU device plugin using a non-root user. To do this,
+the nodes' DAC rules must be configured to device plugin socket creation and kubelet registration.
+Furthermore, the deployments `securityContext` must be configured with appropriate `runAsUser/runAsGroup`.
```

Comment:
Maybe here should also be (e.g. link to) note about container manager (e.g. containerd) config option for whether container device access rights are inherited from the target node host, or from pod security context?

Reply:
moved as a tracking item in https://github.com/intel/intel-device-plugins-for-kubernetes/issues/1045